integrator/integratorlib.py

```diff
@@ -0,0 +1,65 @@
+from subprocess import Popen
+
+
+def find_asteroid(i,f2,lrec,stlen):
+# Find orbital elements for a given asteroid.
+# Given:
+#   i - <string> asteroid name
+#   f2 - file object for the source file
+#   lrec - number of records in the source file
+#   stlen - record length of the source file (must be constant)
+# Returns:
+#   st - <list> the corresponding record to a given asteroid
+#           (if no records found, returns None)
+
+    print('\n\tSeeking record about ',repr(i))
+    st=[]
+    flag=0
+    a=0
+    z=lrec
+    f2.seek(a*stlen)
+    st=f2.readline().split()
+    if st[0] == i:
+        print('\tFound record\n',' '.join(st),'\nat position ',int(a)+1)
+        return st
+    f2.seek((z-1)*stlen)
+    st=f2.readline().split()
+    if st[0] == i:
+        print('\tFound record\n',' '.join(st),'\nat position ',int(z))
+        return st
+    print('\t...Start iterations',a,z)
+    while z-a>1 and flag==0:
+        o=(z+a)//2
+        f2.seek(o*stlen)
+        st=f2.readline().split()
+        if st[0]==i:
+            flag=1
+            break
+        elif st[0]>i:
+            z=o
+        else:
+            a=o
+    if st[0] != i:
+        print('\t...Not found any record about ',repr(i))
+        return None
+    else:
+        print('\tFound record\n',' '.join(st),'\nat position ',int(o)+1)
+    return st
+
+
+def proc(smallin,sheader):
```

Comment:
Please, consider the renaming of the function. I don't understand what "proc" function should do.